sdcm/fill_db_data.py

```diff
@@ -43,7 +43,9 @@ class FillDatabaseData(ClusterTester):
     NON_FROZEN_SUPPORT_OS_MIN_VERSION = '4.1'  # open source version with non-frozen user_types support
     NON_FROZEN_SUPPORT_ENTERPRISE_MIN_VERSION = '2020'  # enterprise version with non-frozen user_types support
     NULL_VALUES_SUPPORT_OS_MIN_VERSION = "4.4.rc0"
+    NULL_VALUES_SUPPORT_ENTERPRISE_MIN_VERSION = "2021.2.rc0"
```

Comment:
What is 2021.2.?

Reply:
If you just need something that is larger than 2021.1 which doesn't support it, you can just the name of the parameter to not supported version and change your if condition.

Reply:
OR change it to 2022.1.0 for now.

Reply:
Ot change it to 2022.1.0 for now.

Reply:
> If you just need something that is larger than 2021.1 which doesn't support it, you can just the name of the parameter to not supported version and change your if condition.

The null query isn't support in 2021.1.rc0, but it's supported in enterprise branch, I saw the latest VERSION was updated to 2021.2.dev by https://github.com/scylladb/scylla-enterprise/commit/2d605a9db6467cdd992b746100bc68ca7040e277

The PR is for master, the change will also be used for upgrade test of enterprise (2021.2.dev).
```
>>> parse_version('2021.1.rc0') > parse_version('2021.1.dev')
True
```

Reply:
@roydahan I just updated the version to 2021.2.dev , can you review again?

If developer changed the version in enterprise later （eg: 2022.1.dev), we also need to update it in SCT.

Reply:
if enterprise branch is 2021.2.dev change it to that.

Reply:
> if enterprise branch is 2021.2.dev change it to that.

Done.